examples/Example8_SendReceiveBinary/Example8_SendReceiveBinary.ino

```diff
@@ -0,0 +1,139 @@
+// Copyright 2023 Blues Inc.  All rights reserved.
+//
+// Use of this source code is governed by licenses granted by the
+// copyright holder including that found in the LICENSE file.
+//
+// This example shows how a developer can efficiently send and receive large
+// binary payloads using the Notecard.
+// https://blues.dev/guides-and-tutorials/notecard-guides/sending-and-receiving-large-binary-objects
+
+#include <stdlib.h>
+#include <Notecard.h>
+#include "helper.h"
+
+// Note that both of these definitions are optional; just prefix either line
+// with `//` to remove it.
+// - Remove txRxPinsSerial if you wired your Notecard using I2C SDA/SCL pins
+//   instead of serial RX/TX
+// - Remove usbSerial if you don't want the Notecard library to output debug
+//   information
+
+HardwareSerial stlinkSerial(PIN_VCP_RX, PIN_VCP_TX);
+// #define txRxPinsSerial Serial1
+#define usbSerial Serial
+
+#include "example_img.h"
+const uint8_t *img_map = example_img_map;
+const size_t img_len = example_img_len;
+
+#define CHUNK_SIZE 8192
+uint8_t temp_buffer[CHUNK_SIZE + 128];
```

Comment:
It would be good to have a comment explaining why the + 128 is needed.

Reply:
I'm struggling with this one. For some reason replacing line 30 with this is causing `NoteBinaryTransmit` to return "buffer is too small for encoding". Do I still need to pad `req_size` with some arbitrary value?

```
size_t req_size = NoteBinaryRequiredTxBuffer(CHUNK_SIZE);
uint8_t *temp_buffer = new uint8_t[req_size];
```

Reply:
It's because of the way the code is structured. I'll hop on a call with you in the morning.

Reply:
I think this is solved now